Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -234,6 +234,13 @@ CookieStore::~CookieStore()
     m_mainThreadBridge->detach();
 }
 
+static String normalize(const String& string)
+{
+    if (string.contains(isTabOrSpace<char16_t>))
+        return string.trim(isTabOrSpace);
```

Comment:
Nit: Perhaps we should specify `<char16_t>` here too for the sake of clarity and consistency with the above line? Not a big deal. Up to you.

Reply:
Fair enough. Is it required in `contains`?

Reply:
Yeah, unfortunately. Not entirely sure why.